google-api-go-generator/gen.go

```diff
@@ -2106,7 +2138,11 @@ func (meth *Method) generateCode() {
 		pn(`})`)
 	}
 	if meth.supportsMediaUpload() && meth.api.Name == "storage" {
-		pn("return gensupport.SendRequestWithRetry(c.ctx_, c.s.client, req)")
+		pn("if c.retry != nil {")
+		pn("	return gensupport.SendRequestWithRetry(c.ctx_, c.s.client, req, c.retry)")
+		pn("} else {")
```

Comment:
Nit: is the `else` necessary if the `if` always returns?

Reply:
Good point, fixed.